docs/components/unstyled/sign-in-button.mdx

```diff
@@ -156,6 +156,13 @@ In some cases you will want to use your own button, or button text. You can do t
         <code>string</code>,
         <>Full URL or path to navigate after successful sign up.</>,
       ],
+    },
+    {
+      cells: [
+        <code>mode</code>,
+        <code>'redirect' | 'modal'</code>,
+        <>Determines what happens when a user clicks on the <code><SignInButton /></code>. Setting this to <code>'redirect'</code> will redirect the user to the sign-in route. Setting this to <code>'modal'</code> will open a modal on the current route.<br />Defaults to <code>'redirect'</code>.</>
```

Comment:
```suggestion
        <>Determines what happens when a user clicks on the <code>{'<SignInButton />'}</code>. Setting this to <code>'redirect'</code> will redirect the user to the sign-in route. Setting this to <code>'modal'</code> will open a modal on the current route.<br />Defaults to <code>'redirect'</code>.</>
```

Reply:
And this is why it needs to be converted to a markdown table 🙈